Cargo.toml

```diff
@@ -12,17 +12,19 @@ readme = "README.md"
 resolver = "2"
 
 [features]
-default = ["u64_backend", "serialize"]
+default = ["rand", "ristretto_u64", "serialize"]
+ristretto_u64 = ["curve25519-dalek/u64_backend"]
+ristretto_u32 = ["curve25519-dalek/u32_backend"]
+ristretto_fiat_u64 = ["curve25519-dalek/fiat_u64_backend"]
+ristretto_fiat_u32 = ["curve25519-dalek/fiat_u32_backend"]
+ristretto_simd = ["curve25519-dalek/simd_backend"]
```

Comment:
I renamed it to signify that it's optional, now P-256 and Ristretto are kinda equal in the naming scheme.